demos/browser/webpack.config.js

```diff
@@ -7,9 +7,11 @@ var HtmlWebpackInlineSourcePlugin = require('html-webpack-inline-source-plugin')
 var HtmlWebpackPlugin = require('html-webpack-plugin');
 /* eslint-enable */
 
-const app = process.env.npm_config_app || 'meetingV2';
-
 module.exports = env => {
+  console.info('Env:', JSON.stringify(env, null, 2));
+  console.info('App:', process.env.npm_config_app);
+  const app = env.app || process.env.npm_config_app || 'meetingV2';
+  console.info('Using app', app);
```

Comment:
Is the index `meetingV2.html` correct? Shouldn't it be correspond to the app here?

Reply:
Unrelated bug, but yes, probably!